sdk/src/Services/DynamoDBv2/Custom/DataModel/TransactWrite.cs

```diff
@@ -466,13 +470,15 @@ public partial class MultiTableTransactWrite
         public MultiTableTransactWrite(params TransactWrite[] transactionParts)
         {
             allTransactionParts = new List<TransactWrite>(transactionParts);
+            TracerProvider = allTransactionParts[0].TracerProvider;
```

Comment:
For defensive code purposes can we check if the collection is empty and if it is use the default trace provider from `AWSConfigs.TelemetryProvider`?

Reply:
Done